src/main/java/org/fundacionjala/sfdc/pageobjects/Contacts/SFCDetailsPage.java

```diff
@@ -0,0 +1,79 @@
+package org.fundacionjala.sfdc.pageobjects.Contacts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SFCDetailsPage extends SalesForceConnection {
+    private WebDriverWait wait;
+
+    @FindBy(how = How.CSS, using = "#topButtonRow > input[name='edit']")
+    private WebElement editContactButton;
+
+    @FindBy(how = How.CSS, using = "#topButtonRow > input[name='del']")
+    private WebElement deleteContactButton;
+
+    @FindBy(how = How.CLASS_NAME, using = "topName")
+    private WebElement newContactLabel;
+
+    /**
+     * Constructor.
+     */
+    public SFCDetailsPage() {
+        super();
+        PageFactory.initElements(this.webDriver, this);
+    }
+
+    /**
+     * Method to get "edit" button.
+     * @return web element edit button.
+     */
+    public WebElement getEditButton() {
+        return this.editContactButton;
+    }
+    /**
+     * Method to get "delete" selected contact.
+     * @return delete button.
+     */
+    public WebElement getDeleteButton() {
+        return this.deleteContactButton;
+    }
+    /**
+     * Method to know if an account is saved.
+     * @return the new contact name displayed in label.
+     */
+    public String newContactSavedName() {
+        return this.newContactLabel.getText();
+    }
+    /**
+     * Method to do click "delete" selected contact.
+     */
+    public void clickDeleteContact() {
+        getDeleteButton().click();
+    }
+    /**
+     * Method to do click accept button on the confirm alert before delete contact.
+     */
+    public void clickDeleteAlert() {
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)